### PR TITLE
source-shopify: ensure inline schemas, updated cdk, poetry (where possible)

### PR DESCRIPTION
**Pull request by bleonard**
This was created from a set of automated scripts. In each case, not every update was needed for every connector, but overall here is what happened:

 * `auto-schema update -s all description`: added descriptions to json and inline schemas
 * `connector-code migrate_to_yaml -c all --type source`: migrates json schemas to connectors with a manifest
 * `airbyte-ci connectors --name=<all modified connectors from above> migrate_to_base_image`: makes sure that each is using a base docker image and updates docs
 * `airbyte-ci connectors --name=<all modified connectors from above> migrate-to-poetry`: moves connectors not already using poetry to do so and updates documentation
 * `airbyte-ci connectors --name=<all modified connectors from above> up_to_date`: updated the CDK to newer (0.80.0) version

The version number and changelogs were also bumped using the `connector-code pulls` command.

### CODE REVIEW

File: airbyte-integrations/connectors/source-shopify/pyproject.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ include = "source_shopify"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "^0"
+airbyte-cdk = "0.81.4"
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: This should not be changed.

**Review by bazarnov — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-shopify/pyproject.toml (hunk 1 above)
Comment (bleonard) [reply airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: It's a change, I understand, but this looks to be better than what's happening with the open pinning. causing less issues.

Comment (bleonard) [reply airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: The actual code in use has not changed.

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: If there are no functional changes, i don't think the source bump should take place either, we can safely merge this without source version bump, these changes than be picked up later on with the next functional change (bug fix or improvements). 

The cdk version pin will not help us later on when we can ant to be able to update the sources with the latest functionality causing to bump each and every source again.

Comment (bleonard) [reply airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: but I also updated all the descriptions

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/pyproject.toml:20]: do we plan to use that in UI?

**Review by maxi297 — APPROVED** [review]
I assume the formatting issue should pass. Can we re-run this? Else, LGTM so this is a conditional approval!

**Review by bazarnov — APPROVED** [review]

**Comment by bazarnov** [comment]
The format check issues should be autoformatted and committed after the `/format-fix`

**Pull request merged**